dada_tests/parser/float_no_decimal.dada

```diff
@@ -0,0 +1,5 @@
+fn main() {
+    a = 1.
```

Comment:
Should we permit this? People sometimes do.

Reply:
I have no opinion.